Tasks/IISWebAppManagementOnMachineGroup/task.loc.json

```diff
@@ -34,6 +34,12 @@
       "visibleRule": "IISDeploymentType = IISWebsite && ActionIISWebsite = CreateOrUpdateWebsite && CreateOrUpdateAppPoolForWebsite = true"
     },
     {
+      "name": "IISWebsiteAuthentication",
+      "displayName": "ms-resource:loc.group.displayName.IISWebsiteAuthentication",
+      "isExpanded": true,
+      "visibleRule": "ConfigureAuthenticationForWebsite = true"
+    },
```

Comment:
Please modify the visibility rule as the current rule will make it visible in other configuration types as well. Caching issues.